### PR TITLE
Adds new cat plants. Alters previous cat plants to better fall in line with lore.

### PR DESCRIPTION
**Pull request by Menown**
- Adds Nm'shaan, Sugar Trees. Produces sugar. Multiharvest.
- Adds n'fri'hi, blizzard ears. Produces flour.
- Alters s'rrendar's hand and messa's tear. Former becomes multiharvest with reduced yield. Latter as well as former both take longer to mature, much like the new plants. All require less water and light, but require lower temperatures.

### CODE REVIEW

**Comment by veraknocks** [comment]
Hopefully the yield of leaves from the Nm'shaan are either one or two units per as generally it would be inefficient/very hard/impossible to grind substantial and viable product from a leaf. Maybe test some mutation strains (Sugar Canopy?) with this later on once you're fully satisfied with your work to boost the depth and value of these new xenoflora specimens. I see you often working the garden and that's about all the Messa's tear that will ever be ploughed for the day. Nice add, buddy.

**Comment by Menown** [comment]
> 
> 
> Hopefully the yield of leaves from the Nm'shaan are either one or two units per as generally it would be inefficient/very hard/impossible to grind substantial and viable product from a leaf. Maybe test some mutation strains (Sugar Canopy?) with this later on once you're fully satisfied with your work to boost the depth and value of these new xenoflora specimens. I see you often working the garden and that's about all the Messa's tear that will ever be ploughed for the day. Nice add, buddy.

You don't process the leaves from the sugar trees. You take the fruit from them, that grows on the top. As said in the lore. This is reflected in-game, as well.

File: html/changelogs/funwithcats.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.  
+author: ChangeMe
```
Comment (Conspiir) [inline html/changelogs/funwithcats.yml]: Forgot to change the author, Meowy.

File: code/modules/hydroponics/seed_datums.dm [hunk 2]
```diff
@@ -1332,4 +1341,50 @@
 	set_trait(TRAIT_PRODUCT_ICON,"bulb")
 	set_trait(TRAIT_PRODUCT_COLOUR,"#ffeedd")
 	set_trait(TRAIT_PLANT_ICON,"stalk")
-	set_trait(TRAIT_WATER_CONSUMPTION, 5)
\ No newline at end of file
+	set_trait(TRAIT_WATER_CONSUMPTION, 5)
+
+/datum/seed/mushroom/nfrihi
+	name = "nfrihi"
```
Comment (Alberyk) [inline code/modules/hydroponics/seed_datums.dm]: Should use the translated name, like the other plants do.

File: code/modules/hydroponics/seed_datums.dm [hunk 3]
```diff
@@ -1332,4 +1341,50 @@
...
+	set_trait(TRAIT_POTENCY,1)
+	set_trait(TRAIT_PRODUCT_ICON,"nfrihi")
+	set_trait(TRAIT_PRODUCT_COLOUR,"#DBDA72")
+	set_trait(TRAIT_PLANT_COLOUR,"#31331c")
+	set_trait(TRAIT_PLANT_ICON,"nfrihi")
+	set_trait(TRAIT_WATER_CONSUMPTION, 4)
+	set_trait(TRAIT_IDEAL_LIGHT, 3)
+	set_trait(TRAIT_IDEAL_HEAT, 253)
+/datum/seed/nmshaan
+	name = "nmshaan"
```
Comment (Alberyk) [inline code/modules/hydroponics/seed_datums.dm]: Should use the translated name, like the other plants do.

File: html/changelogs/funwithcats.yml (hunk 1 above)
Comment (Menown) [reply html/changelogs/funwithcats.yml]: fucc

File: code/modules/hydroponics/seed_datums.dm (hunk 3 above)
Comment (Menown) [reply code/modules/hydroponics/seed_datums.dm]: Nif Berries don't. Only thing that matters is that the display name and the seed packets are translated, which they are. These are just code identifiers.

File: html/changelogs/funwithcats.yml (hunk 1 above)
Comment (Menown) [reply html/changelogs/funwithcats.yml]: resolved...............

**Comment by Menown** [comment]
Closing this for now while I further refine it.

**Pull request closed without merging**